cmd/dep/signals.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
Nah, we can just do it as a separate PR against gps right now. No problem there.